config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreYaml.java

```diff
@@ -16,16 +16,16 @@
 package io.stargate.config.store.yaml;
 
 import com.codahale.metrics.MetricRegistry;
+import com.datastax.oss.driver.shaded.guava.common.annotations.VisibleForTesting;
+import com.datastax.oss.driver.shaded.guava.common.base.Ticker;
+import com.datastax.oss.driver.shaded.guava.common.cache.CacheBuilder;
+import com.datastax.oss.driver.shaded.guava.common.cache.CacheLoader;
```

Comment:
@tatu-at-datastax should we open an issue to replace this with caffeine?

Reply:
Good point, yes.

Reply:
Created: https://github.com/stargate/stargate/issues/1255